skbio/alignment/_alignment.py

```diff
@@ -627,7 +627,9 @@ def update_ids(self, ids=None, fn=None, prefix=""):
                                           "or fn is provided.")
 
         if ids is not None:
-            fn = lambda _: ids
```

Comment:
:cry:

However, from the PEP8 example, maybe we can keep those super simple functions in one line

> Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier. 
Yes:
`def f(x): return 2*x`.